azure-pipelines.yml

```diff
@@ -25,12 +25,40 @@ steps:
   inputs:
     command: 'pack'
     packagesToPack: 'Microsoft.Azure.IoT.Edge.Module.nuspec'
+    packDestination: '$(Build.ArtifactStagingDirectory)/Package'
     versioningScheme: 'off'
 
 - task: AzureArtifacts.manifest-generator-task.manifest-generator-task.ManifestGeneratorTask@0
   displayName: 'SBOM Generation Task'
   inputs:
     BuildDropPath: '$(Build.ArtifactStagingDirectory)'
```

Comment:
>$(Build.ArtifactStagingDirectory) [](http://example.com/codeflow?start=20&length=33)

please update this path to $(Build.ArtifactStagingDirectory)/Package so that the task picks up the package

Reply:
updated